database/raft/rand.go

```diff
@@ -0,0 +1,15 @@
+package raft
+
+import (
+	"crypto/rand"
+)
+
+func randID() []byte {
+	c := 16 // prevent collisions
```

Comment:
Normal people almost never use arrays in Go. (We do quite a bit because we have 256-bit hash floating around all over the place so we got used to them.) So prob the most idiomatic way to write this is `b := make([]byte, 16)`.